Makefile

```diff
@@ -115,6 +115,11 @@ gendependabot:
 		  echo "  - package-ecosystem: \"docker\"\n    directory: \"$${dir:1}\"\n    schedule:\n      interval: \"daily\"" >> ${DEPENDABOT_PATH} ); \
 	done
 
+.PHONY: generate
```

Comment:
Does this target need to be explicitly called during CI?

Reply:
Actually, no.  I'd wager you might have some good thoughts on another Makefile etc to place this in.


that said I could see it advantageous as a "wait you actually reran generate and there's no delta from the commit" check if you think it's wise to add such